vault/external_tests/misc/kvv2_upgrade_test.go

```diff
@@ -24,17 +23,12 @@ import (
 func TestKVv2_UpgradePaths(t *testing.T) {
 	m := new(sync.Mutex)
 	logOut := new(bytes.Buffer)
```

Comment:
Won't this test break since logOut is no longer populated?

Reply:
Hmm, I didn't notice any breakages in my local runs but I'll try it again and put it back if necessary.